docs/transpile/custom_passes.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: Transpilation passes beyond the standard set
+description: Passes not included in the transpiler default optimization levels.
+
+---
+# Custom passes 
+
+Many transpiler passes within Qiskit are optional and are not executed in any of the default optimization levels. In some cases, they can take the place of default passes within those stages. Others can be written outside of Qiskit and executed as plugins. Here we introduce a pass, `PadDynamicalDecoupling`, that adds an error suppression technique called _dynamical decoupling_ to the circuit. Because this pass only operates on scheduled circuits and involves gates that are not necessarily basis gates of our `target`, we will need the `ALAPScheduleAnalysis` and `BasisTranslator` passes as well.
```

Comment:
I like the first 2 sentences of the intro but after that it wasn't clear to me when you say youre "introducing" a pass whether its one that's already available in qiskit or one you are making from scratch. I managed to guess later on by looking at the imports lol but it would be good to make that clearer in the intro paragraph. I'd recommend instead saying something like "In this page we'll be constructing a custom passmanager including built-in Qiskit passes PadDynamicalDecoupling, ALAPScheduleAanalysis and BasisTranslator. This does dynamical decoupling blah blah..."

Reply:
I rewrote this.